static/app/views/eventsV2/table/queryField.tsx

```diff
@@ -144,7 +144,11 @@ class QueryField extends React.Component<Props> {
             fieldValue.function[i + 2] = undefined;
           }
         } else if (param.kind === 'value') {
-          fieldValue.function[i + 1] = param.defaultValue || '';
+          if (param.dataType === 'autoNumber' && param.defaultValue === 'auto') {
+            fieldValue.function[i + 1] = '';
```

Comment:
This is turning `'auto'` into `''` right because the backend is expecting `apdex()` with no arguments right? Can we have a comment explaining this?